pkg/templater/expander.go

```diff
@@ -30,11 +31,25 @@ type AuthService interface {
 	auth.CredentialsCreator
 }
 
+type Phase string
+
+const (
+	PhasePrepare Phase = "prepare"
+	PhaseExpand  Phase = "expand"
```

Comment:
Still relevant?
I think that we agreed that the two steps will be squashed into one, no?